### ISSUE TITLE
Allow zoom for UI elements (including font size)

### ISSUE THREAD

**Issue by luixxiul** [brave/browser-laptop#1937]
From support

> a top priority would be an option to change the font size for the UI. There are a lot of folks using fullHD screens (at minimum) and the dpi settings from windows does not seem to apply to Brave’s UI.
> 
> Could you add this on your priority list as in this moment, it is the only thing that stops me from using Brave (I can’t see the damn tabs – they are so small that I need glasses - or even better a magnifying glass - on a fullHD 15 inch screen)

**Comment by Lennium** [comment]
It would be great to have such a feature, as there are people with eye issues around which are needing to increase UI font size to make the browser usable.

**Comment by bradleyrichter** [comment]
I think we should be able to address this in a different way with resolution independence to fix the usability part. (automatic in Mac OS. Not sure about Windows versions.)

Later we can and normal/large UI fonts in prefs.

**Comment by bsclifton** [comment]
Starting with 0.13.0, 125% DPI setting will actually be respected by Brave 😄

**Comment by bsclifton** [comment]
[Picture shared via Twitter](https://twitter.com/maxart/status/833752426322669568):
![image](https://cloud.githubusercontent.com/assets/4733304/23142405/fc770aa6-f778-11e6-93ff-168178f48492.png)

**_NOTE_**: the user is showing that they'd like to DECREASE the size of the UI

**Comment by bradleyrichter** [comment]
yes, this is from Vivaldi which does exactly what the "bug-now-feature" did from that old PR which needs "feature mining".

**Comment by bsclifton** [comment]
@NejcZdovc since you're working on ReduxComponent optimizations, I'll grab this one 😄

**Comment by MatthewS2077** [comment]
Please someone fix this!
Being able to up-scale the UI is so important for "older" users who are running high DPI displays.
I've only just discovered Brave and I like it a lot BUT not being to up-scale the UI is the one thing that will make me switch back to Vivaldi.
Surely there are thousands of users who are turned off by not being able to up-scale the UI and never bother to complain and just uninstall Brave.

**Comment by luixxiul** [comment]
the latter suggestion is being tracked here: https://github.com/brave/browser-laptop/issues/1757

**Comment by frank6tg** [comment]
UI scaling should be completely separate from content on web page scaling.
